charts/camunda-platform-alpha-8.8/values.yaml

```diff
@@ -77,11 +77,37 @@ global:
     ## @param global.multitenancy.enabled if true, then enable multitenancy in all applicable components.
     enabled: false
 
-  ## Authorizations configuration.
-  ## @extra global.authorizations
-  authorizations:
-    ## @param global.authorizations.enabled if true, then enable authorizations checks in all applicable components.
-    enabled: false
+  security:
+    ## Authentication configuration
+    ## @extra global.security.authentication
+    authentication:
+      ## @param global.authentication.method defines the authentication method which should be used. Possible values: basic, oidc
+      method: basic
+      basic:
+        ## @param global.authentication.allowUnAuthenticatedApiAccess if true, then allow unauthenticated API access.
+        allowUnAuthenticatedApiAccess: false
+    ## Authorizations configuration.
+    ## @extra global.security.authorizations
+    authorizations:
+      ## @param global.authorizations.enabled if true, then enable authorizations checks in all applicable components.
+      enabled: false
```

Comment:
🔧 let's have them enabled by default, as now it's not just about resource based authorizations, like before but also application access etc.